share/docker/Dockerfile

```diff
@@ -106,7 +106,7 @@ RUN pip install zstandard
 RUN pip3 install zstandard
 
 # Install Ruby modules
-RUN gem install echoe yajl-ruby multi_json snappy zstd-ruby
+RUN gem install echoe yajl-ruby multi_json snappy zstd-ruby rubocop
```

Comment:
Instead of maintaining the list of gems in two places, could we install based on the Gemfile?